api/src/main/java/ca/bc/gov/educ/api/graduation/service/RESTService.java

```diff
@@ -50,7 +50,10 @@ public <T> T get(String url, Class<T> clazz, String accessToken) {
             obj = webClient
                     .get()
                     .uri(url)
-                    .headers(h -> { h.setBearerAuth(accessToken); h.set(EducGraduationApiConstants.CORRELATION_ID, ThreadLocalStateUtil.getCorrelationID()); })
+                    .headers(h -> { h.setBearerAuth(accessToken); h.set(EducGraduationApiConstants.CORRELATION_ID, ThreadLocalStateUtil.getCorrelationID());
+                        h.set(EducGraduationApiConstants.HEADER_USER_NAME, ThreadLocalStateUtil.getHeaderUserName());
```

Comment:
Why are we sending username and headerUserName? We only need one username. Ensure all methods only use one username.